### ISSUE TITLE
Accessibility Violation on Trustbar

### ISSUE THREAD

**Issue by Croskm** [itswebny/universal-navigation#234]
The wording on the new UNav trustbar element needs some correction to meet accessibility standards. It currently displays:
"An official website of New York State. Here’s how you know" with "Here's how you know" being the interactive element. That wording is not descriptive enough for users utilizing a screen reader. When it receives focus, it reads as, "Here's how you know. Button. Collapsed." No context as to what it is. A better choice would be to change the wording to something like, "How to identify official NY.gov sites."
Additionally, the item is styled as a hyperlink and not a button.
![Accessibility-violation-unav](https://github.com/user-attachments/assets/77fc8991-c075-4ec2-89cb-d94492756542)
